### DISCUSSION TITLE
arrange by name

### DISCUSSION THREAD

**Discussion by tatobarresi** [rejetto/hfs#354] [Q&A]
![Cattura](https://github.com/rejetto/hfs/assets/145445552/dc69192f-5806-4d71-ae55-0630138c3267)

Hi everyone, how do I correctly arrange by name? in this screen it places the 10 before the 1

**Comment by rejetto** [comment]
ciao,
click on options and enable "numeric names", but also "sort by names"

**Comment by tatobarresi** [comment]
Ciao, ho risolto nominando la prima cartella da "SETTIMANA 1" A "SETTIMANA 01".  grazie mille
